Source/NetworkService.swift

```diff
@@ -46,6 +48,27 @@ public final class NetworkService: NetworkServiceProviding {
         self.networkResponseProcessor = NetworkResponseProcessor()
     }
     
+    /**
+     Fetches a resource asynchronously from remote location. Completion and Error block will be called on the main thread.
+     
+     ```swift
+     
+     let networkService: NetworkServiceProviding = //
+     let resource: Ressource<String> = //
```

Comment:
There's a typo: Resource<String> (only with one s).
This applies also for all other documentation where this is mentioned.